### DISCUSSION TITLE
Korean translation

### DISCUSSION THREAD

**Discussion by FarrellLukas** [vuejs-translations/guidelines#57] [General]
한국어 번역 논의 여기서
[vuejs-translations](https://github.com/vuejs-translations)한국어 관련된 문서가 없어 생성했습니다.
생성후 기 번역된 사이트가 있다는것을 알게되어 추가합니다.
한국어 번역된 문서 사이트 [https://v3-docs.vuejs-korea.org/](https://v3-docs.vuejs-korea.org/)

**Comment by narusas** [comment]
we transalate at https://github.com/vuejs-kr/docs-next 
at ko-KR directory. 

Discussion channel:  vuejs-korea.slack.com

we had completed to translate over 90%.

**Comment by narusas** [comment]
Hi,  @yyx990803
We re-fork to  https://github.com/vuejs-kr/docs 
and Translation is done almost. 
Check  https://v3-docs.vuejs-korea.org/

So we ask you to create official korean transalte repository.

**Comment by narusas** [comment]
Hi, @Jinjiang  
Is that you account for traslation site? 
We has done to translate korean. 
So We ask you to create official repository for korean translation. 
Or Let me know who care about it. 

Thank you :)

**Reply by ryuhangyeong** [reply]
You did a really good job.

**Comment by yyx990803** [comment]
@narusas  Sorry for the late reply! I've created the `kr` team and invited you - please let me know once you've completed the transfer so I can set up the repo / team access.

Also let me know who else to invite to the team. Thank you for your great work!

**Reply by narusas** [reply]
Hi, I transfer repositoroty to https://github.com/vuejs-translations/docs 
I think some mistake at name of repository. 
Please check and fix please. I have no permission.

**Reply by yyx990803** [reply]
Updated! https://github.com/vuejs-translations/docs-kr
You should also have admin access

**Reply by yyx990803** [reply]
I've set up deployment at https://kr.vuejs.org/

It's deployed on Vercel. If you have a Vercel account, please apply to join the `vuejs` team so you can also manage the deployments.

You can also remove the GitHub pages deployment workflow.

**Reply by yyx990803** [reply]
It seems the DNS still needs some time to propagate, in the meanwhile the deployment can be checked at https://docs-kr.vercel.app/

**Reply by narusas** [reply]
Thank. 
Translation guideline said, "use ISO_639-1_codes"
So Is it ko.vuejs.org is correct? instead of kr.vuejs.org . `kr` is national code. `ko` is language code 

(and kr.vuejs.org was  vue2 korean translation link) 
Please check.  
Thank.

**Reply by yyx990803** [reply]
Sorry, I'll update accordingly

**Reply by yyx990803** [reply]
It seems the v2 deployment for kr.vuejs.org was broken due to a domain verification issue (the `vuejs-kr` lost verification of the domain when we verify the root `vuejs.org` domain in the `vuejs` org).

Since I don't have access to the `vuejs-kr` org, I forked the v2 repo at https://github.com/vuejs-translations/v2.ko.vuejs.org so I can deploy it.

Also:

- kr.vuejs.org will now redirect to ko.vuejs.org
- Old v2 pages under `kr.vuejs.org/v2/*` will redirect to `v2.ko.vuejs.org/v2/*`
